src/components/Popular/Popular_section.tsx

```diff
@@ -0,0 +1,30 @@
+// src/pages/LandingPage.tsx
+import { useEffect } from 'react';
+import { useDispatch } from 'react-redux';
+import MostPopular from './MostPopular';
+import MostRecent from './MostRecent';
+import MostSelling from './MostSelling';
+import BannerAD from './BannerAD';
+import { fetchProducts } from '@/features/Popular/availableProductSlice';
+import { AppDispatch } from '../../app/store';
+
+function PopularSection() {
+  const dispatch: AppDispatch = useDispatch();
+
+  useEffect(() => {
+    dispatch(fetchProducts());
+  }, [dispatch]);
+
+  return (
+    <section className=" h-auto md:pl-[30px] md:pr-[20px] md:pb-2 px-[20px]">
+      <BannerAD></BannerAD>
```

Comment:
why don't you use self closing tag here?

Reply:
done